tests/cases/clear_with_depth.vkscript

```diff
@@ -0,0 +1,32 @@
+# Copyright 2018 The Amber Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+[require]
+VK_KHR_get_physical_device_properties2
+depthstencil D32_SFLOAT_S8_UINT
```

Comment:
Since this is the first example for depth-stencil, it's worth adding a comment here saying something like:

```
# Requesting a depthstencil format causes the engine to create a depth-stencil texture and to attach it to the pipeline.
```

Reply:
added